### DISCUSSION TITLE
Is there any kind of audit trail with Leantime?

### DISCUSSION THREAD

**Discussion by danshearer** [Leantime/leantime#581] [Project Management] [answered]
I know everything is logged in the database, but I'm looking for some kind of report.

A commandline script post-processing a database extraction would be fine too, perhaps someone has done one of those?

Dan

**Comment by marcelfolaron** [comment, marked as answer]
This is something we are working on right now. We are logging most events
into the database and should have a simple UI for a full audit trail ready
soon. Next step is to use hyper ledger to store the trail in Blockchain as
an option.

**Reply by danshearer** [reply]
That's good, just what we need. Thanks.

Dan

**Reply by thiagolamosa** [reply]
> This is something we are working on right now. We are logging most events into the database and should have a simple UI for a full audit trail ready soon. Next step is to use hyper ledger to store the trail in Blockchain as an option.
> […](#)
> On Tue, Jan 18, 2022, 9:40 AM Dan Shearer ***@***.***> wrote: I know everything is logged in the database, but I'm looking for some kind of report. A commandline script post-processing a database extraction would be fine too, perhaps someone has done one of those? Dan — Reply to this email directly, view it on GitHub <#581>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ALG4EFTZWVVHVDGQAOTJATDUWV3V3ANCNFSM5MHLD4WA> . Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Which table is used to store these events?
